science/orthanc-dicomweb: Revive port

yzgyyang opened the pull request:
Plugin to bring support of the DICOMweb standard into Orthanc.

PR:		242547
Reviewed by: lwhsu

lwhsu commented on science/orthanc-dicomweb/Makefile:
```diff
@@ -0,0 +1,64 @@
+EXTRACT_ONLY=	OrthancDicomWeb-${PORTVERSION}.tar.gz
+
+MAINTAINER=	maintainer.freebsd@xpoundit.com
+COMMENT=	Plugin to bring support of the DICOMweb standard into Orthanc
```
Just nitpicking, but hope those COMMENTs could align, now they have:
Orthanc plugin
Plugin to bring support ... into Orthanc
Plugin to extend Orthanc

yzgyyang replied:
Nice catch, didn't really pay too much attention to COMMENTs across this ports - should have done.

lwhsu commented on science/orthanc-dicomweb/Makefile:
```diff
@@ -0,0 +1,64 @@
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/Font-Awesome-4.7.0.tar.gz ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/bootstrap-vue-2.0.0-rc.24-dist.tar.gz ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/babel-polyfill-6.26.0.min.js.gz ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/Orthanc-1.9.3.tar.gz ${WRKSRC}/ThirdPartyDownloads
```
Also just nitpicking, if this was me, I would use a `.for` loop for `${DISTFILES}`

lwhsu commented on science/orthanc-dicomweb/pkg-plist:
```diff
@@ -0,0 +1,2 @@
+share/orthanc/plugins/libOrthancDicomWeb.so
+share/orthanc/plugins/libOrthancDicomWeb.so.1.6
```
I think portlint(1) will say this can be put to PLIST_FILES in Makefile.

lwhsu commented on science/orthanc-dicomweb/Makefile:
```diff
@@ -0,0 +1,64 @@
+# Created by: maintainer.freebsd@xpoundit.com
+
+PORTNAME=	orthanc-dicomweb
+DISTVERSION=	1.6
```
I think we can just use `PORTVERSION` here.

yzgyyang replied:
Ah, in my understanding, isn't `DISTVERSION` preferred over `PORTVERSION` (though in this case they are the same anyways)?

lwhsu replied:
TBH I'm a bit confused about this. It seems on in porter's handbook, but most ports are using `PORTVERSION`, anyway, I think keeping `DISTVERSION` is fine.

yzgyyang commented:
@lwhsu Could you review this please? I am not 100% sure about if my creative usage of `EXTRADISTFILES` is correct / if there is a better way.

Commit:
science/orthanc-dicomweb: Revive port

Plugin to bring support of the DICOMweb standard into Orthanc.

PR:		242547
Reviewed by:	lwhsu
